homeassistant/helpers/config_validation.py

```diff
@@ -973,6 +973,56 @@ def custom_serializer(schema: Any) -> Any:
     return voluptuous_serialize.UNSUPPORTED
 
 
+def expand_condition_shorthand(value: Any | None) -> Any:
```

Comment:
Salvage `single_key_schemas` from [this PR](https://github.com/home-assistant/core/pull/66622/files#diff-f9f84381af1676630c88469d5fb4f69f36934019ffbbf187b78f9c976f03cae4R915) and let's use that.

Reply:
I guess that won't work with allowing extra keys

Reply:
There may be more keys, though.

```yaml
alias: "A shorthand or condition"
not:
  - {{ false }}
````

Reply:
For the life of me I'll never understand why github won't show all the comments until I reply to some of them...